test/XmlDataContractSerializerInputFormatterTests.cs

```diff
@@ -1,499 +0,0 @@
-// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
```

Comment:
just curious: these files show up as deletes because you haven't rebased recently?